ast_mapper_class.ml

```diff
@@ -603,6 +603,8 @@ class mapper =
       | PTyp x -> PTyp (this # typ x)
       | PPat (x, g) -> PPat (this # pat x, map_opt (this # expr) g)
       | PSig x -> PSig (this # signature x)
+
+    method constant (c : Parsetree.constant) = c
```

Comment:
If I am not mistaken, the `constant` method is new because there is a location that should be mapped in the `string` case now.